JASP-Engine/JASP/R/reliabilityAnalysisFrequentist.R

```diff
@@ -364,13 +363,14 @@ reliabilityFrequentist <- function(jaspResults, dataset, options) {
   
   cfi <- vector("list", length(boot))
   names(cfi) <- names(boot)
-  
   for (nm in names(boot)) {
-    if (any(is.na(boot[[nm]]))) {
+    if (nm == "mean" | nm == "sd")
```

Comment:
```suggestion
    if (nm %in% c("mean, "sd"))
```

Either this, or use `||`